pyproject.toml

```diff
@@ -22,8 +22,9 @@ filterwarnings = [
 [tool.cibuildwheel]
 # update skip when numpy wheels become available
 skip = ["*-musllinux_*", "cp310-win32", "cp310-manylinux_i686"]
-test-requires = "pytest"
-test-command = "python -m pytest {package}/tests"
+# before-test = "python download_data.py"
```

Comment:
It doesn't work, because, as it is said in docs to cibuildwheel

> A shell command to run in each test virtual environment, **before your wheel is installed** and tested.

So it fails, because `download_data.py` requires `chromo` to be installed. The line will be removed.

Although tests pass on CI without `download_data.py`, I will add to CIBW_TEST_COMMAND.